### ISSUE TITLE
Scaling not working on IE11

### ISSUE THREAD

**Issue by jpmnteiro** [Sitebase/react-avatar#142]
First off, thank you for the component, it's great!

Seems like the `font-size` calculation is not working correctly on Internet Explorer 11.

(I'd add images but seems like I'm unable to do so).

Assuming a `textSizeRatio={2}` and `size="40"`:

- Chrome
  - Font-size property: `calc(120.414%)`
  - Computed font-size: `15.6538px`

- Internet Explorer 11
  - Font-size property: `calc((100% * 2.433098696129971) / 2)`
  - Computed font-size: `2.33px`

IE [has some knowns problems](https://caniuse.com/#search=calc) with calc, so I wonder if the problem is coming from it.

The full avatar element:

```
<div class="sb-avatar sb-avatar--text" style="border-radius: 10%; width: 40px; height: 40px; font-family: Helvetica, Arial, sans-serif; vertical-align: middle; display: inline-block;">
   <div class="ignore-react-onclickoutside cursor-pointer sb-avatar__text" style="background: rgb(66, 133, 244); border-radius: 10%; width: 40px; height: 40px; text-align: center; color: rgb(255, 255, 255); text-transform: uppercase;">
      <div style="width: 100%; height: 100%; display: table;"><span style="font-size: calc((100% * 2.433098696129971) / 2); vertical-align: middle; display: table-cell;"><span>JB</span></span></div>
   </div>
</div>
```

**Comment by JorgenEvens** [comment]
Related to https://github.com/Sitebase/react-avatar/issues/136

**Comment by jpmnteiro** [comment]
@JorgenEvens glad to see this got fixed, thank you 👍 

Do you have a rough idea as to when a release with the fix will be made available?

**Comment by JorgenEvens** [comment]
Hi @jpmnteiro ,

I was waiting for a different PR which has merged this morning.
I'll publish a new release when I get home this evening.

**Comment by JorgenEvens** [comment]
This was fixed in version [v3.5.0](https://github.com/Sitebase/react-avatar/releases/tag/v3.5.0)!
